pkg/build/graph/helpers.go

```diff
@@ -76,3 +78,27 @@ func defaultNamespace(value, defaultValue string) string {
 	}
 	return value
 }
+
+// BuildPointsToTag returns the buildConfig that points to the provided imageStreamTag.
+func BuildPointsToTag(g osgraph.Graph, istag graph.Node) (*buildgraph.BuildConfigNode, bool) {
+	for _, bcNode := range g.PredecessorNodesByEdgeKind(istag, BuildOutputEdgeKind) {
+		return bcNode.(*buildgraph.BuildConfigNode), true
```

Comment:
Why are we just returning the first item in this list? I don't really understand the utility of this method over `isTagNode.PredecessorNodesByEdgeKind`. The name of the method makes me think I'd be accepting a BC/BCNode and IST/ISTNode and returning `true` if the relationship exists in the graph?